data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -64,6 +65,27 @@ def _climate_factor(self) -> bool:
         # Low income: In 60th percentile or above for percent of block group population
         # of households where household income is less than or equal to twice the federal
         # poverty level. Source: Census's American Community Survey]
+
+        # I am thinking that we make constants for these since these thresholds are used
```

Comment:
Yeah, agreed. Let's just create it once. Maybe add a _create_low_income_threshold method that runs before anything else and generates this field, and then we can just reference the field?

Reply:
Yup - my idea exactly